src/operator/tensor/la_op.h

```diff
@@ -81,10 +81,21 @@ struct LaMatrixMultParam : public dmlc::Parameter<LaMatrixMultParam> {
   }
 };
 
+// Parameters for Cholesky factorization and matrix inversion
+struct LaCholeskyParam : public dmlc::Parameter<LaCholeskyParam> {
+  bool lower;
+  DMLC_DECLARE_PARAMETER(LaCholeskyParam) {
+    DMLC_DECLARE_FIELD(lower)
+      .set_default(true)
+      .describe("Triangular matrix is lower triangular.");
```

Comment:
Would it be clearer in the description to say: "'lower' is true if matrix is lower triangular or false for upper triangular" to remove ambiguity?

Reply:
Good suggestion. Done.